gnucash/gnome/gnc-plugin-page-register.c

```diff
@@ -483,7 +484,10 @@ static GtkActionEntry gnc_plugin_page_register_actions [] =
         G_CALLBACK (gnc_plugin_page_register_cmd_exchange_rate)
     },
     {
-        "JumpTransactionAction", GNC_ICON_JUMP_TO, N_ ("_Jump"), NULL,
+/* Translators: This is a menu item that will open a register tab for the
+ * account of the first other account in the current transaction's split list
+ * with focus on the current transaction's entry in that register. */
+        "JumpTransactionAction", GNC_ICON_JUMP_TO, N_ ("_Jump to Other Account"), NULL,
         N_ ("Jump to the corresponding transaction in the other account"),
```

Comment:
Unrelated, but IMHO wrong. Either
"Jump to the corresponding split [of this transaction] in the other account." or
"Show this transaction in the other account."

Reply:
How about "Open a new register tab for the other account with focus on this transaction."?

Reply:
You can also jump between already open register tabs, e.g. forward and back.

Reply:
True. "Jump to register tab of the other account with focus on this transaction."? "Jump to register tab of the other account, opening one if needed, with focus on this transaction."? How wordy do we want to get? The explanation in the [Help Manual section](https://www.gnucash.org/docs/v4/C/gnucash-help/acct-jump.html) isn't very good either.

Reply:
"Jump to register tab of the other account with focus on this transaction."  is sufficient here and "Jump to register tab of the other account, opening one if needed, with focus on this transaction." in help, but I am open for improvements.

There are additional reference sections in Help:
https://www.gnucash.org/docs/v4/C/gnucash-help/ch04s03.html#Trans-menus
https://www.gnucash.org/docs/v4/C/gnucash-help/ch04s03.html#Trans-toolbar
(should get some rework so that we have to define an entry only once)